implementations/micrometer-registry-zabbix/build.gradle

```diff
@@ -0,0 +1,7 @@
+dependencies {
+    compile project(':micrometer-core')
+    compile 'org.slf4j:slf4j-api:1.7.+'
+    compile 'io.github.hengyunabc:zabbix-sender:0.0.5'
```

Comment:
We can make this `latest.release` and rely on dependency locking to always keep the version consistent inside of minor release branches.

Reply:
@matzepan this would be a great addition! I was looking for a possibility to use Zabbix to receive metrics from a Java application and this would neatly solve this issue by enabling me to use micrometer :+1:
Do you still plan to change this line? 
Is there anything else preventing this from getting merged, @jkschneider ?

Reply:
@milgner I changed the line and updated the merge request.